doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -85,9 +89,12 @@ include::../protos/vkGetDeviceProcAddr.txt[]
   ** sname:VkQueue
   ** sname:VkCommandBuffer
 
-If pname:pName is not the name of one of these Vulkan commands, and is
-not the name of an extension command belonging to an extension enabled for
-pname:device, then fname:vkGetDeviceProcAddr will return `NULL`.
+If pname:pName is not the name of one of these Vulkan commands, and is not
```

Comment:
Okay, here "these" refers to any Vulkan command whose first parameter is one of VkDevice, VkQueue or VkCommandBuffer. With the description  immediately above the statement "If pname:pName ..." it's okay.